landlab/components/flow_director/flow_director_steepest.py

```diff
@@ -228,13 +345,330 @@ def direct_flow(self):
+
+        A value of zero indicates that the link does not exist or is not
+        active.
+
+        A value of -1 indicates that water flow based on
+        ``flow__link_to_receiver_node`` goes from head node to tail node, while
+        a value of 1 indicates that water flow goes from tail node to head
+        node.
+
+        Examples
+        --------
+        >>> from landlab import RasterModelGrid
+        >>> from landlab.components import FlowDirectorSteepest
+        >>> mg = RasterModelGrid((3,3))
+        >>> mg.set_closed_boundaries_at_grid_edges(True, True, True, False)
+        >>> _ = mg.add_field('topographic__elevation',
+        ...                  mg.node_x + mg.node_y,
+        ...                  at = 'node')
+        >>> fd = FlowDirectorSteepest(mg, 'topographic__elevation')
+        >>> fd.run_one_step()
+        >>> fd.flow__link_direction_at_node
+        array([[ 0,  0,  0,  0],
+               [ 0, -1,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0, -1],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0]])
+
+        This method will be updated when the DepressionFinderAndRouter is run.
+
+        First, without DepressionFinderAndRouter:
+
+        >>> from landlab.components import FlowAccumulator
+        >>> mg1 = RasterModelGrid(5,5)
+        >>> z1 = mg1.add_field('node', 'topographic__elevation',
+        ...                    mg1.x_of_node+2 * mg1.y_of_node)
+        >>> z1[12] -= 5
+        >>> mg1.set_closed_boundaries_at_grid_edges(True, True, True, False)
+        >>> fa1 = FlowAccumulator(mg1, flow_director='Steepest')
+        >>> fa1.run_one_step()
+        >>> fa1.flow_director.links_to_receiver
+        array([-1, -1, -1, -1, -1,
+               -1,  5, 15,  7, -1,
+               -1, 19, -1, 20, -1,
+               -1, 23, 24, 25, -1,
+               -1, -1, -1, -1, -1])
+        >>> fa1.flow_director.flow__link_direction_at_node
+        array([[ 0,  0,  0,  0],
+               [ 0, -1,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0, -1,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0, -1],
+               [ 0,  1,  0,  0],
+               [ 0,  0,  0, -1],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 1, -1,  0,  0],
+               [-1, -1,  1,  1],
+               [ 0, -1, -1,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0, -1],
+               [ 0,  0,  0, -1],
+               [ 0,  0,  0, -1],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0],
+               [ 0,  0,  0,  0]])
+
+        Next with DepressionFinderAndRouter:
+
+        >>> mg2 = RasterModelGrid(5,5)
```

Comment:
```python
        >>> mg2 = RasterModelGrid((5, 5))
```

Reply:
This has more to do with Issue #864, right? I assume this means you prefer spacing.

Reply:
I got rid of them in all landlab/components/flow_* folders... It will make finding and removing deprecation warnings easier when I eventually deal with #864.